### PR TITLE
use labs theme as external

### PR DESCRIPTION
**Pull request by OriolAbril**
## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [ ] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [ ] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--830.org.readthedocs.build/en/830/

### CODE REVIEW

**Comment by OriolAbril** [comment]
The logo image is too long now and the navbar doesn't fit on mobile screens, I'll have to think how to handle it

File: docs/source/index.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+:align: center
+:class: only-dark
+:::
+
+:::{image} _static/marketing-logo-light.jpg
+:align: center
+:class: only-light
+:::
+
+# PyMC Marketing - Open Source Marketing Analytics Solution
```
Comment (twiecki) [inline docs/source/index.md:11]:
```suggestion
# PyMC-Marketing - Open Source Marketing Analytics Solution
```

**Comment by OriolAbril** [comment]
It should be ready to merge, however, it would be great if we could have 3-4 people check the docs website: https://pymc-marketing--830.org.readthedocs.build/en/830/ from their phones to double check everything looks right, no overflows, elements out of place...

**Comment by juanitorduz** [comment]
> It should be ready to merge, however, it would be great if we could have 3-4 people check the docs website: https://pymc-marketing--830.org.readthedocs.build/en/830/ from their phones to double check everything looks right, no overflows, elements out of place...

Looks great in my phone!

**Comment by juanitorduz** [comment]
@wd60622  or @twiecki, can you give it a look (also your phone) 🙏 If there is no issue, we merge. I just wanna have another tested as suggested :)

**Comment by wd60622** [comment]
Good on my end too! Looks great

**Review by wd60622 — APPROVED** [review]

**Comment by juanitorduz** [comment]
Thank you @OriolAbril 🚀

**Pull request merged**